spec/models/claim_spec.rb

```diff
@@ -642,21 +653,37 @@
       it "adds an error" do
         expect(claim.errors.messages[:base]).to include("You’re not eligible for this payment")
       end
+
+      it "raises an exception" do
+        expect { claim.submit! }.to raise_error(Claim::NotSubmittable)
+      end
     end
 
     context "when the claim has already been submitted" do
       let(:claim) { create(:claim, :submitted, submitted_at: 2.days.ago) }
 
-      it "returns false" do
-        expect(claim.submit!).to eq false
+      it "raises an exception" do
+        expect { claim.submit! }.to raise_error(Claim::NotSubmittable)
       end
 
       it "doesn't change the reference number" do
-        expect { claim.submit! }.not_to(change { claim.reference })
+        expect {
```

Comment:
I'd personally add `RSpec::Matchers.define_negated_matcher :not_change, :change` to `spec_helper.rb` and combine these three specs:

```ruby
expect { claim.submit! }
  .to raise_error(Claim::NotSubmittable)
  .and not_change { claim.reference }
  .and not_change { claim.submitted_at }
```